velox/connectors/hive/storage_adapters/abfs/tests/AzuriteServer.h

```diff
@@ -36,8 +36,8 @@ static const std::string AzuriteAccountKey{
     "Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=="};
 static const std::string AzuriteABFSEndpoint = fmt::format(
     "abfs://{}@{}.dfs.core.windows.net/",
-    AzuriteAccountName,
```

Comment:
Why did this not break anything before?

Reply:
@majetideepak , because AzuriteAccountName and AzuriteContainerName both are `test`